src/core/Text/Header/HeaderText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStyles } from './styles';
+import Typography from '@material-ui/core/Typography';
+
+type Type = 'bold' | undefined;
+
+interface HeaderTextProps {
+  type?: Type;
+  children: string;
+}
+
+const HeaderText: React.FC<HeaderTextProps> = ({ type, children }) => {
+  const classes = useStyles();
+
+  function getType(type: Type): string {
+    switch (type) {
+      case 'bold':
+        return classes.bold;
+      case undefined:
+        return classes.primary;
+    }
+  }
+
+  return <Typography className={`${getType(type)}`}>{children}</Typography>;
```

Comment:
We need to Omit all the default variants of Typograph in `base.ts` since we are providing our own.

Reply:
@S-ayanide That is, we need to only have interface which won't extend it from base.ts ?

Reply:
We won't require the default variant for Typography so we need to omit that